packages/o2-blocks/babel.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+	extends: require.resolve( '@automattic/calypso-build/babel.config.js' ),
```

Comment:
We might be able to import this as a `preset` rather than an `extends` here, since unlike Calypso's top-level `babel.config.js`, we're not overriding any of the settings coming from it here. Then again, is that the precedent we want to set? :thinking: